### PR TITLE
Enable SSR using @uvue/vue-cli-plugin-ssr

### PR DESCRIPTION
**Pull request by davidyuk**
Screenshot 2020-12-06 at 20 54 42

[Prefetch api](https://universal-vue.github.io/docs/guide/usage.html#prefetch-mixin) is the best

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -2,4 +2,5 @@ module.exports = {
   presets: [
     '@vue/cli-plugin-babel/preset',
   ],
+  plugins: ['@babel/plugin-proposal-optional-chaining'],
```
[the remark this answers is not in the document]
Comment (davidyuk) [reply babel.config.js:5]: It already works fine in vue cli and used in some places. Probably it is a fix for the SSR backend process.

[the remark this answers is not in the document]
Comment (davidyuk) [reply babel.config.js:5]: also, this is explained in the commit message: https://github.com/aeternity/superhero-ui/pull/908/commits/bab0de88dd9802c15ba953ff7b237c1922aeeda5

**Comment by davidyuk** [comment]
I would merge this after fixing #909

**Comment by davidyuk** [comment]
depends on #1139 fixes #586 #1007

proposing to display page render time in the footer:

Screenshot 2021-07-14 at 01 12 10

that to check in SSR version:
- navigate between pages with disabled JS (to ensure that there are no errors in rendering on server side and it's time is acceptable, less than 4s, 0.3s on average)
- ensure that everything works as in develop with JS enabled (without extra exceptions in console), try to reload pages while testing to ensure that app works correctly based on pages rendered by server

**Review by Liubov-crypto — COMMENTED** [review]
UI + web wallet:
Trending - pass 
Search - pass 
Change profile pic- pass 
Change back pic  - pass 
Change profile info - pass 
Cookies - pass 
Send Tip on the Feed - pass
Retip on the feed - pass 
Post without tipping - pass  
Add comment - pass 
Tip comment on the feed - pass 
Pin/Unpin - pass 
Claim - pass 
Report post - pass 
Voting - pass 
Vote - pass 
Revoke Vote - pass 
Delegate voting power - pass 
Create a new poll - pass 
Search - pass 
Meet - pass 
Updates - pass 
Faq- pass 

When searching in the feed, the rendering time will be longer than the PR suggests:
search time

If I reload the page, the post goes out of the feed and becomes larger, and render time is more than 0.4s (if the feed is loaded for a long time, then when the page is reloaded, the rendering time will be more than 0.4 seconds):

https://user-images.githubusercontent.com/69896204/125636976-5321fab1-4de3-4a63-a590-b0c27ce8f483.mov

When I reload the page, the connection between UI and the wallet constantly disappears, I need to re-accept the pop-up for connection.

**Comment by davidyuk** [comment]
> When searching in the feed, the rendering time will be longer than the PR suggests

I think this is acceptable, it depends on backend response time

Also, every first request in 15 minutes would take ~1-1.5 seconds due to the server reloading cached chain names, tip info, and token prices.

> If I reload the page, the post goes out of the feed and becomes larger

This is a tricky thing, it looks like that browser starts to display page earlier than it receives whole markup and styles. Probably it may be solved by reordering columns making the left and right columns go before the tip list only in markup (for example, using [order](https://developer.mozilla.org/en-US/docs/Web/CSS/order) property). I don't think that this is crucial, we can proceed as it is now.

> When I reload the page, the connection between UI and the wallet constantly disappears, I need to re-accept the pop-up for connection.

This is expected behavior. Currently solvable by permissions on the wallet side.

**Review by AtanasKrondev — APPROVED** [review]
Great work!

**Pull request merged**